config/prod.exs

```diff
@@ -9,7 +9,7 @@ config :logger, level: :info
 
 # run the server automatically when using prod release
 config :jellyfish, JellyfishWeb.Endpoint,
-  http: [ip: {0, 0, 0, 0, 0, 0, 0, 0}, port: 8080],
+  http: [ip: {127, 0, 0, 1}, port: 8080],
```

Comment:
Why do we change it? Most of the time, we run prod in docker, so changing it to 127.0.0.1 doesn't make much sense.

Reply:
For docker we override this variable to 0.0.0.0 and because we run Jellyfish behind nginx (there is no way to configure SSL directly in Jellyfish right now), I think it is safer not to expose Jellyfish to the outside by default